test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -527,5 +534,8 @@ private static async Task SendGitHubPullRequestComments(int prNumber, string bod
 
             response.EnsureSuccessStatusCode();
         }
+
+        private static Policy Retry(int retryCount = 5)
+            => Policy.Handle<HttpRequestException>(r => r.StatusCode == HttpStatusCode.ServiceUnavailable).WaitAndRetry(retryCount, retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)));
```

Comment:
When git.exe failed, it didn't raise `HttpRequestException`.

Reply:
Only retry when 503